01 SFC Hello VueJS/README.md

```diff
@@ -0,0 +1,264 @@
+# 01 SFC Hello VueJS
+
+## Why SFC?
+
+> https://vuejs.org/v2/guide/single-file-components.html
+
+In many Vue projects, global components will be defined using `Vue.component`, followed by `new Vue({ el: '#container' })` to target a container element in the body of every page.
+
+This can work very well for small to medium-sized projects, where JavaScript is only used to enhance certain views. In more complex projects however, or when your frontend is entirely driven by JavaScript, these disadvantages become apparent:
+
+- **Global definitions** force unique names for every component
```

Comment:
This disadvantages is about using vue with compiler mode using `template` isn't it? (Without JSX syntax)

About "no build step", you could add build steps and using es6 / typescript. What do you mean with that?

Reply:
About "no build step", this is an extract from the official page. If I am honest, I do not finish understanding the reason for the affirmation.